### ISSUE TITLE
Filter options lack value

### ISSUE THREAD

**Issue by torgeirl** [devilry/devilry-django#1208]
WCAG 2.1 Success Criterion 2.4.6 (Headings and Labels) requires that headings and labels have a described topic or purpose.

Some filter options lack values for their default value, and are there hard to navigate using input assistance tools. Some of these could be named «All», but it depends on the filter.
